src/lucky/cookies/cookie_jar.cr

```diff
@@ -1,6 +1,6 @@
 class Lucky::CookieJar
   MAX_COOKIE_SIZE         = 4096
-  LUCKY_ENCRYPTION_PREFIX = Base64.encode("lucky") + "--"
+  LUCKY_ENCRYPTION_PREFIX = Base64.strict_encode("lucky") + "--"
```

Comment:
`Base64.encode` adds a newline at the end, but newline characters are invalid cookie values.